openstates/sd/people.py

```diff
@@ -0,0 +1,125 @@
+import re
+
+from pupa.scrape import Person, Scraper, Organization
+
+import lxml.html
+
+
+class SDLegislatorScraper(Scraper):
+    latest_only = True
+
+    def scrape(self, chambers=None):
+        url = 'http://www.sdlegislature.gov/Legislators/default.aspx' \
+              '?CurrentSession=True'
+        if chambers is None:
```

Comment:
The `scrape` method seems to accept `chamber` instead of `chambers` most of the time. I'm guessing we should keep this consistent.